tools/cbmc/proofs/SendEventToIPTask/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+jobos: ubuntu16
+cbmcflags: '=--object-bits;7;--32;--bounds-check;--pointer-check;--nondet-static='
```

Comment:
Is there a specific reason, why you don't run ```make cbmc-batch.yaml``` for this one?
I would expect ```--unwinding-assertions``` as one of these flags, if you generate this file.
As we are considering generating the cbmc-batch.yaml files as part of the CI in the future, it would be interesting, whether you left this flag out on purpose.

Reply:
I didn't leave it out on purpose, but after regenerating cbmc-batch.yaml it has appeared.